File: eTuristickaAgencija.Models/Uposlenik.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTuristickaAgencija.Models
{
    public class Uposlenik
    {
        public int Id { get; set; }
        public int? KorisnikId { get; set; }
        public DateTime DatumZaposlenja { get; set; }
        public bool? Aktivan { get; set; }
    }
}

File: UI/eturistickaagencija_admin/lib/models/uposlenik.dart
import 'package:json_annotation/json_annotation.dart';

part 'uposlenik.g.dart';

@JsonSerializable()
class Uposlenik{
int? id;
int? korisnikId;
DateTime? datumZaposlenja;
bool? aktivan;

Uposlenik(this.id,this.korisnikId,this.datumZaposlenja,this.aktivan);

factory Uposlenik.fromJson(Map<String,dynamic>json)=>_$UposlenikFromJson(json);
Map<String,dynamic>toJson()=>_$UposlenikToJson(this);
}
